src/main/java/pl/uj/edu/JImageStream/api/ImageStream.java

```diff
@@ -44,6 +44,13 @@ public StreamableImage collect() {
         return new StreamableImage(imageCopy);
```

Comment:
We don't really need this method anymore, we should delete this and refactor Runner using new collector interface